tests/test_upgrade_shared.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
+
+from helpers import run_pipx_cli
+
+
+def test_upgrade_shared(pipx_ultra_temp_env, capsys, caplog, monkeypatch):
+    from pipx.shared_libs import shared_libs
```

Comment:
Should this be moved to the top?

Reply:
I will add a comment in order to explain why it's not at the top.

Reply:
I see, thank you.